test/functional/apps/discover/_collapse_expand.js

```diff
@@ -18,21 +20,23 @@ import {
         var fromTime = '2015-09-19 06:31:44.000';
         var toTime = '2015-09-23 18:31:44.000';
 
-        // start each test with an empty kibana index
-        return scenarioManager.reload('emptyKibana')
+        // delete .kibana index
+        return esClient.delete('.kibana')
+        .then(function () {
+          // wait for Kibana to re-create it and add UTC and defaultIndex
+          return common.try(function () {
```

Comment:
this seems to be a common enough occurrence, do you think it's worth wrapping this with a method that overwrites the kibana index with a dump that sets default configs and saved objects?